tube_server:
  ros__parameters:

    F_star: 3000.0
    d_star: 1.0
    alpha: 10.0
    spring_const: 2000.0
    damp_const: 10.0

    num_obstacles: 8
    obstacles_x: [1.746, -0.311, 0.298, 1.746, -1.192, 2.254, -1.192, -1.683]
    obstacles_y: [-3.746, 0.013, 2.706, 5.042, -3.086, 1.029,  1.893,  3.950]             
    obstacles_h: [1.524, 2.667, 1.524, 1.905, 1.905, 1.524, 1.524, 1.905]
    obstacles_w: [0.508, 0.508, 0.508, 0.508, 0.508, 0.508, 1.016, 0.508]
    obstacles_d: [0.508, 0.508, 1.016, 0.508, 0.508, 0.508, 0.508, 0.508]                   

    num_vertiports: 3
    vertiports_x: [-1.0,  0.5, -0.5, 1.0]
    vertiports_y: [-4.0, -4.0,  4.0, 3.5]
    takeoff_height: 1.0

    num_walls: 6
    walls_a: [0.0, 0.0, 1.0, -1.0,  0.0,  0.0]
    walls_b: [0.0, 0.0, 0.0,  0.0,  1.0, -1.0]
    walls_c: [1.0,-1.0, 0.0,  0.0,  0.0,  0.0]
    walls_d: [0.0, 4.0, 2.5,  2.5,  4.5,  4.5]

    bounds_x: [-1.5, 1.5]
    bounds_y: [-3.5, 3.5]
    bounds_z: [1.5, 2.5]